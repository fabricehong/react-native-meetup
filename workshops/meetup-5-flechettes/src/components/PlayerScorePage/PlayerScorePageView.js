import React, {PureComponent} from 'react';
import {View} from 'react-native';

export class PlayerScorePageView extends PureComponent {
  render() {
    return (
      <View/>
    );
  }
}

export default PlayerScorePageView;