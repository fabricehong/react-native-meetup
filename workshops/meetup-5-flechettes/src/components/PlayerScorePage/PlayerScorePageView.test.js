import React from 'react';
import renderer from 'react-test-renderer';
import PlayerScorePageView from './PlayerScorePageView';

describe('PlayerScorePageView', () => {
  test('should renders correctly', () => {
    const tree = renderer.create(<PlayerScorePageView/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});