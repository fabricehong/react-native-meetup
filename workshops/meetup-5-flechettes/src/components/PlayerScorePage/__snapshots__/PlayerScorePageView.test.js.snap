// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerScorePageView should renders correctly 1`] = `<View />`;
